code/modules/lighting/effects/effects_drugs.dm

```diff
@@ -0,0 +1,25 @@
+/obj/screen/plane/drugs
+	icon = 'icons/effects/256x256.dmi'
+	plane = EFFECTS_PLANE
+	blend_mode = BLEND_MULTIPLY
+	alpha = 80
+	mouse_opacity = 0
+	var/severity // runtime avoidance
+
+/obj/screen/plane/drugs/rainbow
+	icon_state = "cloud"
+
+/obj/screen/plane/drugs/rainbow/New()
+	..()
+	processing_objects |= src
```

Comment:
Might as well use `+=` here. Something very odd would have to be going on for it to already be in `processing_objects`